build-support/docker/travis_ci_py27_ucs2/Dockerfile

```diff
@@ -21,10 +21,12 @@ RUN yum install sqlite-devel -y
 ENV PYENV_ROOT "${PYENV_ROOT:-/pyenv-docker-build}"
```

Comment:
@cosmicexplorer can you please remove lines 19 and 20: `RUN yum install sqlite-devel -y`. We don't need them anymore.